verification.py

```diff
@@ -2220,6 +2220,47 @@ def verify_breakpoint_distribution(
         pyplot.savefig(path)
         pyplot.close('all')
 
+    def verify_recombination(
+            self, basedir, name, sample_size, Ne, r, L, model, growth_rate=0):
+        """
+        Verifies that the number of recombination equals the number of mutation.
+        """
+        if not os.path.exists(basedir):
+            os.mkdir(basedir)
+        empirical_theta = []
+        empirical_rho = []
+        for i in range(1,200):
+                ts = msprime.simulate(
+                    Ne=Ne, recombination_rate=r, mutation_rate=r ,length=L,
+                    population_configurations=[
+                    msprime.PopulationConfiguration(
+                    sample_size=sample_size, initial_size=Ne,
+                    growth_rate=growth_rate)],
+                    model=model)
+                empirical_theta.append(ts.get_num_sites())
+                ts = msprime.simulator_factory(
+                    Ne=Ne, recombination_rate=r,length=L,
+            	    population_configurations=[
+                    msprime.PopulationConfiguration(
+                    sample_size=sample_size, initial_size=Ne,
+                    growth_rate=growth_rate)],
+            	    model=model)
+                ts.runs()
+                empirical_rho.append(ts.num_breakpoints)
+        empirical_rho.sort()
+        empirical_theta.sort()
+        pyplot.plot(empirical_rho, empirical_theta)
```

Comment:
We should use a qqplot here rather than the line plot. See the other examples on how to do this.

Reply:
I edited the code to do a real qqplot. Can you see it on the  pull request ?